README-install.md

```diff
@@ -152,6 +147,10 @@ determine using ansible, you may need to change the following fields in
   passwordless ssh
 * `ansible_become` - use `no` if `ansible_ssh_user` is `root`, otherwise,
   use `yes`
+* `openshift_logging_mux_namespaces` - **REQUIRED** Represents the environment
```

Comment:
According to the line 191, REQUIRED may be a little too strong?
It'd be easier to understand if the line 191-192 is moved next to the line 153?